apps/api/src/queries/buildViewModelForResource/viewModels/base-resource.view-model.ts

```diff
@@ -1,5 +1,6 @@
+import { ContributorWithId } from '@coscrad/api-interfaces';
```

Comment:
We should have a standalone test for term queries at this point. It's an e2e test like `digital-text-queries.e2e.spec.ts`. Make sure to test that contributions come through there thoroughly.

Reply:
built e2e suite for fetch by id and fetch many.